.travis.yml

```diff
@@ -5,28 +5,33 @@ os:
   - linux
   - osx
 
+git:
+  depth: 10
```

Comment:
Doesn't seem to make any difference to speed so I don't think it's worth carrying the extra cruft in .travis.yml

Reply:
Done.